TFT/src/User/API/parseACK.c

```diff
@@ -1,6 +1,6 @@
 #include "parseACK.h"
 #include "includes.h"
-#include "parseACKJson.hpp"
+#include "RRFParseACK.hpp"
```

Comment:
move this include in inludes.h

Reply:
I'm not a fan of this change, I feel that scope should be minimized, and only parseACK.c should have access to anything in RRFParseACK.

Reply:
no problem if the scope will be limited only on parseAck